lib/phoenix/live_dashboard/components/chart_component.ex

```diff
@@ -50,7 +50,14 @@ defmodule Phoenix.LiveDashboard.ChartComponent do
         <div class="card-body">
           <!-- We don't add a phx-update="stream" because js expects only new data -->
```

Comment:
If possible, could this comment be improved? (I'm not sure how it should be written, though.)